### ISSUE TITLE
how I could force the firefox plugin to check in french ?

### ISSUE THREAD

**Issue by JOduMonT** [languagetool-org/languagetool#5182]
Hi;

My browser is in english, the website I visit is in french the text I'm typing is in french but LanguageTools try to check the text in english and when I switch to french it only says Oops!!

At the beginning it want to replace Créer by Creep then avec by AVE
![image](https://user-images.githubusercontent.com/5204724/121504724-6e11a300-ca0c-11eb-9abf-8df3dab152ac.png)

When I switch in French it only say Oops!!
![image](https://user-images.githubusercontent.com/5204724/121504637-5d612d00-ca0c-11eb-803d-f9a7cae7bef5.png)

**Comment by danielnaber** [comment]
Does your text contain more than one language? That situation isn't properly supported yet, but we're working on a fix. If it happens with only French, could you send the text? Your screenshots seem to show that there's markup involved, this might confuse LanguageTool.

**Comment by JOduMonT** [comment]
yes I have English also, while, I thought it would be possible to enfore a language by default. ;)
